server/controllers/list.js

```diff
@@ -308,29 +319,31 @@ const updateListItem = (req, resp) => {
   List.findOneAndUpdate(
     {
       _id: listId,
-      'items._id': itemId,
-      $or: [{ ownerIds: req.user._id }, { memberIds: req.user._id }]
+      'items._id': sanitize(itemId),
+      $or: [{ ownerIds: userId }, { memberIds: userId }]
```

Comment:
Is there a way to change this line to `memberIds: userId` or this is fixed in another task/branch?